cms/djangoapps/contentstore/views/component.py

```diff
@@ -30,6 +30,11 @@
 from django.utils.translation import ugettext as _
 from models.settings.course_grading import CourseGradingModel
 
+try:
+    from xblock_django.models import XBlockDeprecatedAdvancedComponentConfig
+except ImportError:
+    XBlockDeprecatedAdvancedComponentConfig = None
```

Comment:
Yes, I understand. However, `XBlockDisableConfig` is used in places where Django is not available during tests. Hence the need for this pattern there. Since we are using `XBlockDeprecatedAdvancedComponentConfig` in a Django app, it is also going to be accessible.